examples/gaussian_process/plot_compare_gpr_krr.py

```diff
@@ -102,6 +102,7 @@
       % (time.time() - stime))
 
 # Plot results
+plt.figure(figsize = (10,5))
```

Comment:
I am not sure we should change the figsize. Not sure what are the assumptions for the example gallery / carousel.